src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -167,6 +169,15 @@ const ContentListTable = () => {
   const introspectRepoForUuid = (uuid: string): Promise<void> =>
     introspectRepository({ uuid: uuid, reset_count: true } as IntrospectRepositoryRequestItem);
 
+  const { mutateAsync: triggerSnapshotMutation } = useTriggerSnapshot(queryClient);
+  triggerSnapshotMutation;
```

Comment:
This line doesn't do anything, this is referencing the function "triggerSnapshotMutation" you just made, but isn't calling it. 

We also wouldn't want to call it here.